terraform/rtmp-server/bootstrap-post-dns.sh

```diff
@@ -17,15 +17,14 @@ systemctl stop nginx.service
 certbot certonly -n --agree-tos --standalone --email "${EMAIL_ADDRESS}" -d "${DOMAIN_NAME}" -d "ipfs-server.${DOMAIN_NAME}" -d "ipfs-gateway.${DOMAIN_NAME}"
 
 # Configure nginx with HTTPS
-cp -f /tmp/ipfs-server/nginx-default /etc/nginx/sites-available/default
-sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g" /etc/nginx/sites-available/default
+cp -f /tmp/rtmp-server/nginx-default  /usr/local/nginx/conf/conf.d/default
+sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g"  /usr/local/nginx/conf/conf.d/default
 
-cp -f /tmp/ipfs-server/nginx-gateway "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}"
-sed -i "s#__DOMAIN_NAME__#${DOMAIN_NAME}#g" "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}"
-ln -s "/etc/nginx/sites-available/ipfs-gateway.${DOMAIN_NAME}" "/etc/nginx/sites-enabled/ipfs-gateway.${DOMAIN_NAME}"
-
-systemctl start nginx.service
+cp -f /tmp/rtmp-server/nginx-gateway "/usr/local/nginx/conf/conf.d/ipfs-gateway.${DOMAIN_NAME}"
```

Comment:
We need to print the URL of this gateway in terraform provision complete

Reply:
Issue #91